File: npocham.ts
/**
 * Check whether Npocham can use keigo (honorific language / んぽちゃむは敬語を使えるのか確認する)
 */
function can_use_keigo(): string {
    return "無理ちゃむ";
}

export { can_use_keigo };

// Run when executed directly: npx ts-node npocham.ts
if (require.main === module) {
    console.log(can_use_keigo());
}

File: npocham.cpp
#include <iostream>
#include <string>

/**
 * Check whether Npocham can use keigo (honorific language / んぽちゃむは敬語を使えるのか確認する)
 */
std::string can_use_keigo() {
    return "無理ちゃむ";
}

int main() {
    std::cout << can_use_keigo() << std::endl;
    return 0;
}
